CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Contributing
 
 This project adheres to the [Spotify FOSS Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code.
 
-[code-of-conduct]: https://github.com/spotify/medusa/blob/master/CODE_OF_CONDUCT.md
+[code-of-conduct]: ../CODE_OF_CONDUCT.md
```

Comment:
@adejanovski Do we want to use a _hard_ link here and put the absolute path to the CoC, or leave it relative?

Reply:
I think we should keep relative paths, but it's in the same directory as the CONTRIBUTING.md file, so the link should just be `CODE_OF_CONDUCT.md`, right?

Reply:
You're right, not sure how I missed this....